### PR TITLE
add instruction on adding custom font style within Readme

### PR DESCRIPTION
**Pull request by AlexYoungZ**
## Issue
* [Jira Issue Link](https://siyangzhang.atlassian.net/browse/TTG-69)

## Type of change
- This change requires a documentation update

## Description
* Add instruction on adding custom aboriginal font style

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,9 @@
     <li><a href="#on-a-container">On a container</a></li>
   </ul>
 </li>
+<li>
```
Comment (dishant9397) [inline README.md:30]: Lets move this section to before "How to run"?

File: README.md [hunk 2]
```diff
@@ -90,6 +93,12 @@ docker run -p 3000:3000 -it turingtestgame
 
 **Note:** Redirect to [http://localhost:3000](http://localhost:3000) to view it in your browser. The page will not automatically reload when you make the changes because the application is deployed within docker and you might have to run above commands again to see the changes in effect. For further details on docker image, could refer to this guide [here](https://code.visualstudio.com/docs/containers/quickstart-node).
 
+
+## How to upload custom font style
+* Select [Noto Sans Canadian Aboriginal](https://fonts.google.com/noto/specimen/Noto+Sans+Canadian+Aboriginal) font style
```
Comment (dishant9397) [inline README.md:98]: Should we keep something generic? "Select the font that you want to have from ''some reference link here'"?

Comment (AlexYoungZ) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (AlexYoungZ) [reply README.md]: done

**Review by EranXu — APPROVED** [review]
lgtm

File: README.md [hunk 3]
```diff
@@ -90,6 +97,10 @@ docker run -p 3000:3000 -it turingtestgame
 
 **Note:** Redirect to [http://localhost:3000](http://localhost:3000) to view it in your browser. The page will not automatically reload when you make the changes because the application is deployed within docker and you might have to run above commands again to see the changes in effect. For further details on docker image, could refer to this guide [here](https://code.visualstudio.com/docs/containers/quickstart-node).
 
+
```
Comment (dishant9397) [inline README.md:100]: Can we remove these blank lines from this PR? it seems unnecessary.

Comment (AlexYoungZ) [reply README.md]: done

**Review by dishant9397 — APPROVED** [review]

**Pull request merged**